README.md

```diff
@@ -117,6 +117,7 @@ encoded public key for RSA and ECDSA.
 * `ignoreNotBefore`...
 * `subject`: if you want to check subject (`sub`), provide a value here
 * `clockTolerance`: number of seconds to tolerate when checking the `nbf` and `exp` claims, to deal with small clock differences among different servers
+* `clockTimestamp`: the time in seconds that should be assumed as the current time for all necessary comparisons (also against `maxAge`)
```

Comment:
I think you would need to explicitly say something like `also against maxAge, be sure you don't use milliseconds in maxAge when you use both together, since clockTimestamp works with seconds`